org.gnome.gnome-latex.json

```diff
@@ -0,0 +1,124 @@
+{
+  "app-id" : "org.gnome.gnome-latex",
+  "runtime" : "org.gnome.Sdk",
+  "runtime-version" : "3.30",
+  "platform-extensions" : [
+    "org.freedesktop.Sdk.Extension.texlive"
```

Comment:
Have you tested if this works?

I thought this would work like this:

```
sdk-extensions:
- org.freedesktop.Sdk.Extension.texlive
finish-args:
- --env=PATH=/usr/bin:/app/bin:/app/bin/x86_64-linux
modules:
- name: texlive
  buildsystem: simple
  build-commands:
- /usr/lib/sdk/texlive/install.sh
```

Reply:
Yes, Alexander is right here, nothing from texlive is available in the app this way. SDK extensions aren't meant to be package dependencies.